Formula/libhttpseverywhere.rb

```diff
@@ -34,8 +34,11 @@ class Libhttpseverywhere < Formula
   end
 
   def install
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["libsoup@2"].opt_lib/"pkgconfig"
+    ENV.prepend_path "XDG_DATA_DIRS", Formula["libsoup@2"].opt_share
+
     mkdir "build" do
-      system "meson", *std_meson_args, ".."
+      system "meson", *std_meson_args, "-Dintrospection=false", ".."
```

Comment:
What's the error before you tried passing `-Dintrospection=false`?

Reply:
it couldn't find various gir files

Reply:
That's annoying. It's sadly a required build dep tho